### PR TITLE
- templatize default file

### PR DESCRIPTION
**Pull request by cybacolt**
- fix config.sls to match
- add example to pillar.example

improvements regarding issue https://github.com/saltstack-formulas/bind-formula/issues/43 and PR https://github.com/saltstack-formulas/bind-formula/pull/44.

changes revert gracefully now on omission.

### CODE REVIEW

File: pillar.example [hunk 1]
```diff
@@ -13,6 +13,9 @@ bind:
     options:
       allow-recursion: '{ any; };'  # Never include this on a public resolver
 
+   # force bind to serve only one protocol. omitting this reverts to binds default of both.
```
Comment (gravyboat) [inline pillar.example]: Can we update this line to say something like `force bind to serve only one IP protocol (ipv4: 4, ipv6: 6)...`

**Comment by gravyboat** [comment]
@cybacolt I added one line note just so we can try to make this clear for any user since I think the simple use of 4/6 without a bit more explanation might be confusing, other than that this is an awesome change! Let me know what you think about the documentation thing and I'll get this merged!

**Comment by gravyboat** [comment]
@cybacolt Also I don't know if you saw this: https://hacktoberfest.digitalocean.com/ but if you want a free shirt I think you've hit the requirement for October. Not sure if it's still open, but a nice bonus for doing these!

**Comment by cybacolt** [comment]
doco change done, signed up for a free tshirt, but dont like my chances ! :)

**Comment by gravyboat** [comment]
Awesome thanks!

**Pull request merged**